terraform/variables.tf

```diff
@@ -34,6 +34,8 @@ variable key_vault_infra_secret_name {}
 
 variable azure_credentials { default = null }
 
+variable "disable_routes" { default = null }
```

Comment:
It would be more natural to make it a boolean. ie `true` instead of `"true"` and `false` instead of `null`

Reply:
changed to a boolean